bin/start-collectd

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+: ${GRAPHITE_HOST:=}
+: ${GRAPHITE_PORT:="2003"}
+
+set -eu
+
+CONF_TEMPLATE=/etc/collectd/collectd.conf.tpl
+CONF_DEST=/etc/collectd/collectd.conf
+
+HOSTNAME=$(wget -O - --tries 1 --timeout 0.2 http://169.254.169.254/1.0/meta-data/instance-id 2>/dev/null || hostname)
+
+if [ -n "$GRAPHITE_HOST" ]; then
+    sed -e "s/{{ hostname }}/$HOSTNAME/g" \
+        -e "s/{{ graphite_host }}/$GRAPHITE_HOST/g" \
+        -e "s/{{ graphite_port }}/$GRAPHITE_PORT/g" "$CONF_TEMPLATE"
+else
+  sed -e "s/{{ hostname }}/$HOSTNAME/g" \
+      -e "s/LoadPlugin write_graphite/LoadPlugin write_log/g" \
```

Comment:
Ah, I didn't know that supervisor doesn't try to restart a process if it exits with 0 or 2. But, I had to set the `startsecs` down to 0 as well, because:

> Even if a process exits with an “expected” exit code (see `exitcodes`), the start will still be considered a failure if the process exits quicker than `startsecs`.

This is much better 👍